ui/pages/Onboarding/OnboardingAddWallet.tsx

```diff
@@ -32,8 +32,18 @@ export default function OnboardingAddWallet(): ReactElement {
           </div>
           <SharedButton
             type="primary"
-            linkTo="/onboarding/importMetamask"
+            isDisabled
```

Comment:
`isDisabled` should really also be tied to `procss.env.HIDE_ADD_SEED === "true"`.

```suggestion
            isDisabled={procss.env.HIDE_ADD_SEED === "true"}
```

But not going to block this for a dev tweak obv.